.codesandbox/deploy.mjs

```diff
@@ -39,8 +40,11 @@ export async function createCodeSandbox(appPath) {
   );
   // omit linked package
   if (packageJSON.dependencies.fabric.startsWith('file:')) {
-    packageJSON.dependencies.fabric = '*';
+    const { repo, branch } = getGitInfo();
+    packageJSON.dependencies.fabric = `${repo}#${branch}`;
```

Comment:
installs from github 
great for PRs
I guess CodesandboxCI handles this under the hood but for now this is great